deeprank/generate/DataGenerator.py

```diff
@@ -33,7 +35,7 @@ def _printif(string, cond): return print(string) if cond else None
 class DataGenerator(object):
 
     def __init__(self, pdb_select=None, pdb_source=None,
-                 pdb_native=None, pssm_source=None,
+                 pdb_native=None, pssm_source=None, align=None,
```

Comment:
It lacks docstring for `align`.

Reply:
done

Reply:
add one more example for interface alignment?